DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateCourses/DelegateCoursesViewModelTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateCourses
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateCourses;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class DelegateCoursesViewModelTests
+    {
+        private readonly IEnumerable<CourseStatisticsWithAdminFieldResponseCounts> courses =
```

Comment:
Can we use NBuilder here instead?

Reply:
Then you can put it in test code without it cluttering them.

Reply:
I asked this in a different PR, seemed like NBuilder wasn't quite sufficient. Link: https://github.com/TechnologyEnhancedLearning/DLSV2/pull/920#discussion_r803573116